Forms/Tools/SeedCalculator.cpp

```diff
@@ -113,7 +123,20 @@ void SeedCalculator::search35()
     ui->progressBar->setRange(0, searcher->getMaxProgress());
 
     auto *timer = new QTimer();
-    connect(timer, &QTimer::timeout, [=] { ui->progressBar->setValue(searcher->getProgress()); });
+    auto startTime = time(0);
+    connect(timer, &QTimer::timeout, [=] {
+        auto progress = searcher->getProgress();
+        ui->progressBar->setValue(progress);
+        auto elapsedTime = time(0) - startTime;
+        auto estimatedTime = elapsedTime * (searcher->getMaxProgress() - progress) / progress;
+        ui->progressLabel->setText(tr("elapsed time: %1:%2:%3 - estimated time: %4:%5:%6")
```

Comment:
Would prefer the 'e' in elapsed and estimated to be capitalized

Reply:
done bf3a7cbef904324c8530bcfb10da652e9cf58acc